vtex/utils/transform.ts

```diff
@@ -1208,6 +1209,7 @@ export function toPlace(
       openingHoursSpecification: pickupPoint.businessHours?.map(
         toHoursSpecification,
       ),
+      isActive: pickupPoint.isActive ?? false,
```

Comment:
Don't assume that a missing isActive is false. Let the client decide it. Forward the undefined/null isActive from pickupoint.